### DISCUSSION TITLE
Hold middle mouse button and drag for spaces swap

### DISCUSSION THREAD

**Discussion by lambegraham** [linearmouse/linearmouse#396] [Ideas]
Hi,

Currently LinearMouse has every function I am looking for except one. When in fullscreen in an app, I use another app called 'Magic Mouse Fix' to enable me to scroll out of the fullscreen app to my desktop. To do this I hold the middle mouse button and swipe left to right on my screen and it brings me out of the fullscreen app to my desktop. This is an incredibly useful feature to work with fullscreen apps and may consider being implemented in the future?

**Comment by lujjjh** [comment]
Thank you for your suggestion! I've added it to my backlog.

Although gesture support would be a great addition, it's unlikely that it will be implemented soon due to higher priority tasks.

Please track its progress in issue #397.
